py/desitarget/sv3/sv3_cuts.py

```diff
@@ -1252,65 +1262,107 @@ def isBGS_colors(rfiberflux=None, gflux=None, rflux=None, zflux=None, w1flux=Non
     g = 22.5 - 2.5*np.log10(gflux.clip(1e-16))
     r = 22.5 - 2.5*np.log10(rflux.clip(1e-16))
     z = 22.5 - 2.5*np.log10(zflux.clip(1e-16))
+    w1 = 22.5 - 2.5*np.log10(w1flux.clip(1e-16))
     rfib = 22.5 - 2.5*np.log10(rfiberflux.clip(1e-16))
 
     # Fibre Magnitude Cut (FMC) -- This is a low surface brightness cut
     # with the aim of increase the redshift success rate.
     fmc |= ((rfib < (2.9 + 1.2 + 1.0) + r) & (r < 17.8))
     fmc |= ((rfib < 22.9) & (r < 20.0) & (r > 17.8))
+    # Remove next line?
     fmc |= ((rfib < 2.9 + r) & (r > 20))
 
     bgs &= fmc
-
+        
+    # Make sure we grab all the Large Galaxies
+    bgs |= ((LX) & (imaging_mask(maskbits, bgsmask=True)))
+    
+    # D. Schlegel - ChangHoon H. color selection to get a high redshift success rate
+    schlegel_color = (z - w1) - 3/2.5 * (g - r) + 1.2
+    rfibcol =(rfib < 20.75) | ((rfib < 21.5) & (schlegel_color > 0.))
+
+    #BASS r-mag offset with DECaLS
+    offset = 0.025
+    
     if targtype == 'bright':
-        bgs &= rflux > 10**((22.5-19.5)/2.5)
+        if south:
+            bgs &= rflux > 10**((22.5-19.5)/2.5)
+            bgs &= rflux <= 10**((22.5-12.0)/2.5)
```

Comment:
I'd think we want bright-end cuts much fainter than `12.0`. People have generally been suggesting more like `> 16` (for extended sources) and using `FIBERTOTFLUX` instead of `FLUX`. I'm not sure what's appropriate for high-surface-brightness galaxies like the `BGS_BRIGHT` sample in `BRIGHT` time, though. @araichoor and @deisenstein might want to comment.

Reply:
16 seems to aggressive, by doing so we'll be removing ~14 /deg^2 of which 10 /deg^2 are SGA galaxies. What about a cut in r and rfibermag, `r > 12 & rfibmag > 15`, this will remove only 0.3 /deg^2 of which 0.06 /deg^2 are SGA.
![image](https://user-images.githubusercontent.com/33119308/112727558-a400ba00-8ee8-11eb-9d2a-203b26ba37b5.png)

Reply:
Let's adopt that for now and then make a final decision next week. I think the issue is that the 10/deg^2 we'd remove at the bright-limit that are SGA galaxies could saturate anyway, so there's no realistic way to observe them.

I'm happy to merge with your chosen `r > 12 & rfibmag > 15` but you should ask on-list (specifically the operations mailing list) what is a realistic bright-end limit to adopt.

Reply:
Ah, I see @deisenstein made a comment as I was writing this. can you use rfibertotmag (the mag based on rfibertotflux) instead? So `r > 12 & rfibtotmag > 15`?

Reply:
Hi @geordie666, is this `rfibtotmag` available in sv3? I cannot find it.

Reply:
https://github.com/desihub/desitarget/blob/master/py/desitarget/sv3/sv3_cuts.py#L1749-L1761

Reply:
@qmxp55: The `rfibertotflux` column is available, but it's a little tricky to pass it across the different iterations of sv. For now, let's just use your cut of `r > 12 & rfibmag > 15`. I'm currently working on a different branch, and I'll add passing `rfibertotflux` around in _that_ branch.

Let me know when you've added the `r > 12 & rfibmag > 15` cut and I'll merge this PR.

Reply:
I just reviewed the code carefully again, and it looks like you already included the `r > 12 & rfibmag > 15` cut. Thanks!